packages/extension/src/newtab/DndModal.tsx

```diff
@@ -1,95 +1,104 @@
-import React from 'react';
-import classnames from 'classnames';
-import {
-  getDefaultLink,
-  CustomTime,
-  TimeFormat,
-  getExpiration,
-  getTimeFormatExpiration,
-} from './dnd';
-import { Radio } from '../../../shared/src/components/fields/Radio';
-import { Dropdown } from '../../../shared/src/components/fields/Dropdown';
-import { TextField } from '../../../shared/src/components/fields/TextField';
-import { ModalCloseButton } from '../../../shared/src/components/modals/ModalCloseButton';
+import React, { FC, useState, useContext } from 'react';
+import { format } from 'date-fns';
+import { Radio } from '@dailydotdev/shared/src/components/fields/Radio';
+import { Dropdown } from '@dailydotdev/shared/src/components/fields/Dropdown';
+import { TextField } from '@dailydotdev/shared/src/components/fields/TextField';
+import { ModalCloseButton } from '@dailydotdev/shared/src/components/modals/ModalCloseButton';
 import {
   StyledModal,
   ModalProps,
-} from '../../../shared/src/components/modals/StyledModal';
-import styles from './DoNotDisturbModal.module.css';
+} from '@dailydotdev/shared/src/components/modals/StyledModal';
+import { getDefaultLink, dndOption, CustomTime, TimeFormat } from './dnd';
 import DndContext from './DndContext';
 
-export interface DoNotDisturbModalProps extends ModalProps {
-  defaultTimeFormat?: TimeFormat;
-}
-
-const timeFormatOptions = Object.values(TimeFormat).map((value) => ({
-  label: value,
-  value,
+const timeFormatOptions = Object.entries(dndOption).map(([k, v]) => ({
+  label: v.label,
+  value: k,
 }));
-
 const customTimeOptions = Object.values(CustomTime);
 
-const DoNotDisturbModal: React.FC<DoNotDisturbModalProps> = ({
-  defaultTimeFormat = TimeFormat.HALF_HOUR,
+const DoNotDisturbModal: FC<ModalProps> = ({
```

Comment:
Please make sure to use the same name for both the component and the file.
Also our style guide for declaring components is as follows:
`export default function ComponentName(): ReactElement`